examples/logreg.py

```diff
@@ -49,9 +49,9 @@ def generate_fake_training_data():
     FakeInputProvider('/job:localhost/replica:0/task:0/device:CPU:5')
 ]
 
-server0 = tfe.protocol.Server('/job:localhost/replica:0/task:0/device:CPU:0')
-server1 = tfe.protocol.Server('/job:localhost/replica:0/task:0/device:CPU:1')
-crypto_producer = tfe.protocol.CryptoProducer('/job:localhost/replica:0/task:0/device:CPU:2')
+server0 = tfe.protocol.Player('/job:localhost/replica:0/task:0/device:CPU:0')
```

Comment:
You can probably refactor all the example files

Reply:
done